### ISSUE TITLE
Description change.

### ISSUE THREAD

**Issue by NamedKitten** [carykh/jumpcutter#38]
change vidx to videos?

**Comment by UltraNiels** [comment]
Yes, i saw this typo in the initial commit and it was fixed in `README.MD` but not in de description.

**Comment by kool601** [comment]
Maybe vidx is intentional and is referring to the codec. I'm not sure...
